robot/control/Src/motion-control/VelocityKalmanTuner/README.md

```diff
@@ -0,0 +1,65 @@
+# Velocity Kalman Tuner
+## Introduction
+Each of the RoboCup robots uses a Velocity Kalman Filter (RobotEstimator.cpp) in order to estimate linear x-axis 
+and y-axis velocities from noisy encoder data and angular z-axis velocity from noisy gyro data. 
+
+### What is a Kalman Filter? (Finish later)
+In brief, a Kalman Filter 
+
+| Matrix | Name | Size | Description |
+| ------ | --- | ---- | ----------- |
+| **A** | **State Transition Matrix** | 3x3 | Maps current state to next state, assuming dynamics hold
+| **B** | **Control Matrix** | 5x5 | Maps control input to change in state
+| **P** | **Estimate Covariance Matrix** | 3x3 | Correlation between each state variable |
+| **H** | **Observation Matrix** | 4x3 | Maps current states to measurements |
+| **Q** | **Process Covariance Matrix** | 3x3 | Covariance of state variable uncertainty due to unmodeled influences |
+| **R** | **Observation Covariance Matrix** | 5x5 | Covariance of measurement uncertainty due to sensor noise and other influences  | 
+The Velocity Kalman Tuner tool allows the user to tune the Q and R gains of a robot's Velocity Kalman Filter
+
+For more detailed and better understanding, here are a few useful sources:
+- https://www.bzarg.com/p/how-a-kalman-filter-works-in-pictures/
+- https://en.wikipedia.org/wiki/Kalman_filter
+- https://www.youtube.com/playlist?list=PLX2gX-ftPVXU3oUFNATxGXY90AULiqnWT
+
+## Using the Tool
+###Installation
+This tool was last tested with Python 3.7 and requires the following external libraries
+- matplotlib
+- numpy
+- PyQt5
```

Comment:
Python has requirements.txt to automatically get packages at specific versions so you don't have to do everything manually. It's probably ok in this case since these are pretty famous packages and there's only 3, but if versions are very important or there's a lot of them, the requirements.txt is a god send